approved_tags.yaml

```diff
@@ -0,0 +1,103 @@
+---
+- "accessibilità"
+- "ia"
+- "android"
+- "api"
+- "apl"
+- "arte"
+- "domanda"
+- "assembly"
+- "audio"
+- "libro"
+- "browsers"
+- "c"
+- "c++"
```

Comment:
farne una sola c/c++?

Reply:
C e C++ sono linguaggi molto diversi nonostante spesso si collassino in una sola dicitura, io preferirei tenerle separate.

Reply:
va bene